packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.spec.js

```diff
@@ -4,16 +4,32 @@ import sinon from 'sinon'
 
 import InteractionLayer from './InteractionLayer'
 
-const onPointerDown = sinon.stub()
-const onPointerMove = sinon.stub()
-const onPointerUp = sinon.stub()
-
-let wrapper
-
 describe('Component > InteractionLayer', function () {
+  let wrapper
+  const mockMark = {
+    setCoordinates: sinon.stub()
+  }
+  const mockDrawingTask = {
+    activeToolIndex: 0,
+    activeTool: {
+      createMark: sinon.stub().callsFake(() => mockMark)
+    }
+  }
+  const mockSVGEvent = {
+    matrixTransform: sinon.stub().callsFake(() => ({
+      x: 100,
+      y: 200
+    }))
+  }
+  const mockSVG = {
```

Comment:
Is there a better way to create an SVG element for these tests?

Reply:
I'll give that a go.

Reply:
That gives a type error: `TypeError: svg.createSVGPoint is not a function`, which I'm also getting in the storybook for #1308.

Reply:
D'oh. `document.createElement` creates a `HTMLElement` so, yeah, that won't have the SVG interfaces. It'll probably work if I add the appropriate namespace for `svg`.

Reply:
`document.createElementNS` with the SVG namespace doesn't work either.